packages/datadog_tracking_http_client/CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Unreleased
 
+## 2.2.1
```

Comment:
Leave it under the `Unreleased` heading.  We'll add the version when we release it.

Reply:
done